i2c/README.md

```diff
@@ -89,15 +82,15 @@ correct formatted frame for your sensor.
 In these files you can find the implementation of the hardware abstraction layer used
 by Sensirion I2C embedded drivers. This part of the code is specific to the underlying
 hardware platform, therefore multiple implementations exist. We also provide an
-unimplemented file for the user to implement himself or herself.
+unimplemented template for the user to implement.
```

Comment:
This sounds much clearer to me compared to my previous comment. Can we write a similar instruction there?